ir_datasets/docs/hc4.yaml

```diff
@@ -0,0 +1,40 @@
+_:
+  pretty_name: 'HC4: A New Suite of Test Collections for Ad Hoc CLIR'
+  desc: '
+<p>
+HC4 is a new suite of test collections for ad hoc Cross-Language Information Retrieval (CLIR), 
+with Common Crawl News documents in Chinese, Persian, and Russian, topics in English and in 
+the document languages, and graded relevance judgments.
+</p>
+<p>
+To access the docuemnts of this dataset, you will need to download the documents from Common Crawl. 
+The script for downloading and validating the documents are in <a href="https://github.com/hltcoe/HC4">HLTCOE/HC4</a>. 
+Please use the following command to download the documents:
+<code>
+git clone https://github.com/hltcoe/HC4<br/>
+cd HC4<br/>
+pip install -r requirements.txt<br/>
+python download_documents.py --storage ~/.ir_datasets/hc4/ \ <br/>
+                             --zho ./resources/hc4/zho/ids.jsonl.gz \ <br/>
+                             --fas ./resources/hc4/fas/ids.jsonl.gz \ <br/>
+                             --rus ./resources/hc4/rus/ids.*.jsonl.gz \ <br/>
+                             --jobs {number of process} \ <br/>
+                             --check_hash <br/>
+</code>
+You can also store the documents in other directory and create a soft link for <kbd>~/.ir_datasets/hc4/</kbd>.
+</p>
+
+<ul>
+  <li>Documents: Web pages from Common Crawl in Chinese, Persian, and Russian.</li>
+  <li>
+  Queries: English TREC-style title/description queries. Narrative field contains an example passage for each relevance level.
+  Human and machine translation of the titles and descriptions in the target language (i.e., document language) are provided in the query object. 
+  (Titles and descriptions are machine-translated into all three target languages even in the laguages that they are not assessed to facillate CLIR 
+  other than English-to-X pairs, e.g., Persian-to-Chinese. Please refer to the original dataset repository for these additional resources.)
+  </li>
+  <li>Report: Each query comes with an English report that is designed to be written by professional searchers prior to the search.</li>
+  <li>Qrels: Documents are judged in three levels of relevance. Please refer to the dataset paper for the full definition of the levels. </li>
+  <li><a href="https://github.com/hltcoe/HC4">Repository</a></li>
+  <li><a href="https://arxiv.org/abs/2201.09992">Dataset Paper</a></li>
+</ul>'
+  bibtex_ids: ['Lawrie2022HC4']
```

Comment:
Can you add documentation for each subset? (Doesn't need to be as detailed, but something to clarify the language used, etc.)

Reply:
Thanks, Sean!